networks/remote/ansible/roles/add-lcd/templates/gaiacli.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
Might be worthwhile renaming this? This seems to start the LCD, should we rename this to `lcd.service` and related nomenclature?

Reply:
The naming was deliberate because this is supposed to be temporary. A CLI should not act as a server, the LCD functionality needs to be broken out to somewhere else.

I keep naming the services based on the binary name (I used to do it differently and ran into issues).

The more immediate reason why this should not be renamed right now is that the scripts were tested and there's no easy "refactor" button for ansible scripts. If I rename it, I'll have to go through all scripts and check if I ever used it. Everything has to be re-tested again to make sure 1. I didn't break anything and 2. it was updated everywhere. I'd rather leave it like this and move away from it completely as soon as the LCD functionality was separated.
(A more obscure case of the naming is in a template for logz.io so it's not just the tasks you have to update but everything that could depend on the name.)

Reply:
Fair enough.